packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -0,0 +1,94 @@
+# Toolbar
+The toolbar contains multiple components that allow a user to manage and manipulate a data set. Data can be presented in a variety of views (table, list, card, etc.). PatternFly supports multiple toolbar variations that address a variety of use cases.
+
+## Elements
+![toolbar](img/toolbar.png)
+
+The following elements may be included in a typical toolbar.
```

Comment:
You can substitute "can" for "may" and use contractions to make it feel less formal. So, "The following elements can be used in a typical toolbar." 

Also, I wasn't sure what a data presentation component would be, so maybe clarifying with an example? "Bulk selection can be added when the functionality isn't available from the component you're using to present the data. For example, you might need to include a bulk selector for a card view or data list, but a table will enable bulk selection by default." 

"Bulk selection allows the user to select or deselect all items on a page and provide an indeterminate state that notifies the user when only some items are selected. Bulk selectors should always be the left-most element in a toolbar. When providing a bulk selector in a list view, it should be aligned with the checkboxes.